manage-data/migrate/migrate-with-logstash.md

```diff
@@ -0,0 +1,119 @@
+---
+navigation_title: Migrate Elastic Cloud Hosted data to Serverless with Logstash
+applies_to:
+  stack: ga
+  deployment:
+    ess: ga
+    ece: ga
+products:
+  - id: elasticsearch
+  - id: logstash
+  - id: cloud-hosted
+  - id: cloud-enterprise
+  - id: cloud-kubernetes
+---
+
+# Migrate {{ech}} data to {{serverless-full}} with {{ls}} [migrate-with-ls]
+
+[{{ls}}](logstash://reference/index.md) is a data collection engine that uses a large ecosystem of [plugins](logstash-docs-md://lsr/index.md) to collect, process, and forward data from a variety of sources to a variety of destinations. Here we focus on using the [Elasticsearch input](logstash-docs-md://lsr/plugins-inputs-elasticsearch.md) plugin to read from your {{ech}} deployment, and the [Elasticsearch output](logstash-docs-md://lsr/plugins-outputs-elasticsearch.md) plugin to write to your {{{serverless-full}} project.
+
+Familiarity with {{ech}}, {{es}}, and {{ls}} is helpful, but not required. 
+
+:::{admonition} Basic migration
+This guide focuses on a basic data migration scenario for moving static data from an {{ech}} deployment to a {{serverless-full}} project. 
+
+Dashboards, visualizations, pipelines, templates, and other {{kib}} assets must be migrated separately using the {{kib}} [export/import APIs](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-saved-objects) or recreated manually.
```

Comment:
Not just separately, but templates, data stream definitions, ILM policies, should be in place before the migration begins, while visual components can be deployed afterwards.

Reply:
@jsvd, clarification to Kibana assets is in. Dd you want to see this again before I merge it?